index.js

```diff
@@ -1 +1,115 @@
-// implement your API here
+const express = require('express');
+const db = require('./data/db.js');
+const cors = require('cors');
+const server = express();
+server.use(express.json());
+server.use(cors())
+
+server.get('/', (req, res) => {
+  res.send('Go to /api/users to grab that sweet user info.');
+});
+
+server.get('/api/users', (req, res) => {
+  db.find()
+  .then(users => {
+    res.status(200).json(users)
+  })
+  .catch(err => {
+    res.status(500).json({error: "The users information could not be retrieved."})
+  })
+})
+
+server.get('/api/users/:id', (req, res) => {
+  const id = req.params.id
+  db.findById(id)
+  .then(user => {
+    if (!user || user.length < 1) {
```

Comment:
Excellent use of checking against the length of the user. By doing so, you ensure that whole data must be present and any empty values caused by user error are not counted in your CREATE, READ, and DELETE methods. The same logic can be applied to your UPDATE as well.